src/game/game.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+#include "src/generator/generator.h"
+#include "src/game/tile.h"
+
+#define BOARD_WIDTH 20
+#define BOARD_HEIGHT 20
+
+class Game{
+private:
+    std::vector<std::vector<tile>> board;
```

Comment:
One of the discussions about the model class addition:

Excract this to separate model class in models directory. I'd imagine we will have World class which would handle all the logic beetwen different representation of data in the future